channeldb/db.go

```diff
@@ -286,12 +286,6 @@ var (
 			number:    31,
 			migration: migration31.DeleteLastPublishedTxTLB,
 		},
-		{
-			// Create a top level bucket which holds information
-			// about our node announcement.
-			number:    32,
-			migration: mig.CreateTLB(nodeAnnouncementBucket),
```

Comment:
if we remove this here, then no need to add it in the first place :)

Reply:
I initially included this, but after @Chinwendu20 's  review, I realized it was unnecessary. Therefore, I removed it here.

Reply:
Good job @Abdulkbk, you can delete the commit, then :) You can reach out to me on slack if you need help with it